src/app/ipv/middleware.js

```diff
@@ -319,11 +319,6 @@ async function handleUnexpectedPage(req, res, pageId) {
 }
 
 module.exports = {
-  renderAttemptRecoveryPage: async (req, res) => {
-    res.render(getIpvPageTemplatePath(PAGES.PYI_ATTEMPT_RECOVERY), {
-      csrfToken: req.csrfToken(),
```

Comment:
This is the same as handleJourneyPage for PYI_ATTEMPT_RECOVERY

<img width="684" alt="Screenshot 2024-08-05 at 09 33 06" src="https://github.com/user-attachments/assets/09eff94a-55f0-4d4d-8696-4493e4e6b364">